components/filters.py

```diff
@@ -38,5 +38,4 @@ def filter_queryset(self, request, queryset, view):
         return (
                 get_objects_for_user(user, permission, queryset, accept_global_perms=False)
                 .union(queryset.filter(status=Component.Status.PUBLIC))
-                .order_by("pk")
```

Comment:
I thought about this some more, and I didn't think this was the best place for this.